packages/experimental/src/smart-session/ui/views/w3m-smart-session-list-view/index.ts

```diff
@@ -0,0 +1,253 @@
+import { CoreHelperUtil } from '@reown/appkit-core'
+import { customElement } from '@reown/appkit-ui'
+import { LitElement, html } from 'lit'
+import { state } from 'lit/decorators.js'
+import styles from './styles.js'
+import { SmartSessionsController } from '../../../controllers/SmartSessionsController.js'
+import { DateUtil } from '@reown/appkit-common'
+import type { SmartSession } from '../../../utils/TypeUtils.js'
+
+const TABS = 3
+const TABS_PADDING = 48
+const MODAL_MOBILE_VIEW_PX = 430
+const SMART_SESSION_TABS = [{ label: 'Active' }, { label: 'Expired' }, { label: 'Revoked' }]
+@customElement('w3m-smart-session-list-view')
```

Comment:
Could separate the blocks with empty line
```suggestion

@customElement('w3m-smart-session-list-view')
```